module/renderer/src/androidUnitTest/kotlin/org/cru/godtools/shared/renderer/CoilTestImages.kt

```diff
@@ -5,4 +5,7 @@ import coil3.asImage
 
 object CoilTestImages {
     val bruce = Drawable.createFromStream(this.javaClass.getResourceAsStream("bruce.jpg"), "bruce.jpg")!!.asImage()
+    val blackPanther =
+        Drawable.createFromStream(this.javaClass.getResourceAsStream("black_panther.png"), "black_panther.png")!!
+            .asImage()
```

Comment:
unfortunately this is using Android APIs, otherwise I would love to move these into the common test resources file